db/migrate/20240104161831_add_provider_attributes.rb

```diff
@@ -0,0 +1,24 @@
+class AddProviderAttributes < ActiveRecord::Migration[7.1]
+  def change
+    create_enum :provider_type, %w[scitt lead_school university]
+
+    change_table :providers, bulk: true do |t|
+      t.enum :provider_type, enum_type: "provider_type", null: false
+      t.string :name, null: false
+      t.string :ukprn
+      t.string :urn
+      t.string :email
```

Comment:
Schools have the attribute `email_address`, whereas Providers have `email`.

I feel like we should probably standardise on one or the other, to avoid confusing weirdness in our app later down the line. Right now, both attribute names have been influenced by the external data sources, so it seems reasonable to try and leave that complexity at the front door.

(Also relevant to PR #90)

Reply:
If we decide to go with `email` instead of `email_address`, we'll need a ticket to change the field in the school model. This comment is just to remind us to do that.

Reply:
resolved in [Refactor importer to align with GiasImporter](https://github.com/DFE-Digital/itt-mentor-services/pull/92/commits/95894fbc58176adec76570e9578c60a49761f274)